### PR TITLE
Update README after release 0.7.0

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,21 +91,18 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
...
-| otel.config.max.event.attrs         | OTEL_CONFIG_MAX_EVENT_ATTRS        | unlimited      | Maximum number of attributes per event.                                                                                                                                                                                                                                                                                                                                                                   |
-| otel.config.max.link.attrs          | OTEL_CONFIG_MAX_LINK_ATTRS         | unlimited      | Maximum number of attributes per link.                                                                                                                                                                                                                                                                                                                                                                    |
```
Comment (mateuszrzeszutek) [inline README.md:108]: Max attr length, max event & link attributes are no longer configurable by property in the SDK so I've removed them.

**Review by iNikem — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -91,21 +91,18 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
 
 | System property                   | Environment variable              | Description                                                                                                         |
 |-----------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------------------------------|
-| otel.exporter=jaeger-thrift-splunk | OTEL_EXPORTER=jaeger-thrift-splunk              | Select the Jaeger exporter. This is the default value.                                                                                          
+| otel.trace.exporter=jaeger-thrift-splunk | OTEL_EXPORTER=jaeger-thrift-splunk              | Select the Jaeger exporter. This is the default value.
```
Comment (jkwatson) [inline README.md:94]:
```suggestion
| otel.trace.exporter=jaeger-thrift-splunk | OTEL_EXPORTER=jaeger-thrift-splunk              | Select the span exporter to use. `jaeger-thrift-splunk` is the default value.                                                                                          
```

File: README.md [hunk 3]
```diff
@@ -91,21 +91,18 @@ A simple wrapper for the Jaeger exporter of [opentelemetry-java](https://github.
...
 |-----------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------------------------------|
-| otel.exporter=jaeger-thrift-splunk | OTEL_EXPORTER=jaeger-thrift-splunk              | Select the Jaeger exporter. This is the default value.                                                                                          
+| otel.trace.exporter=jaeger-thrift-splunk | OTEL_EXPORTER=jaeger-thrift-splunk              | Select the Jaeger exporter. This is the default value.                                                                                          
 | otel.exporter.jaeger.endpoint     | OTEL_EXPORTER_JAEGER_ENDPOINT     | The Jaeger endpoint to connect to. Default is `http://localhost:9080/v1/trace`.
-| otel.exporter.jaeger.service.name | OTEL_EXPORTER_JAEGER_SERVICE_NAME | The service name of this JVM instance. Default is `unknown`.                                                        
+| otel.exporter.jaeger.service.name | OTEL_EXPORTER_JAEGER_SERVICE_NAME | The service name of this JVM instance. Default is `unknown_service:java`.
```
Comment (jkwatson) [inline README.md:96]:
```suggestion
| otel.exporter.jaeger.service.name | OTEL_EXPORTER_JAEGER_SERVICE_NAME | The service name of this JVM instance. Default is `unknown_service:java`, but any `service.name` provided via other means will supercede this value.
```

**Review by jkwatson — APPROVED** [review]
👍

**Pull request merged**